src/index.js

```diff
@@ -1,11 +1,11 @@
 import app from './app.js';
 import express from 'express';
-import route from './routes/index'
+import route from './routes/index';
 import 'dotenv/config';
 
 // const app = express();
 
-app.use(express.urlencoded({ extended: true }));
+app.use(express.urlencoded({extended: true}));
 app.use(express.json());
 
 route(app);
```

Comment:
You script problem is being caused by this line. Please remove it because you don't need need it anyway.